test/e2e/validation_test.go

```diff
@@ -370,6 +420,21 @@ func (tc *testContext) sshSetup() error {
 	return nil
 }
 
+// ensureTestRunnerRBAC creates the RBAC resources required for the test runner service account
+func (tc *testContext) ensureTestRunnerRBAC() error {
```

Comment:
This function is duplicating https://github.com/jrvaldes/windows-machine-config-operator/blob/4bbebad3cc022d157e62d9b2578be289ef960e9d/test/e2e/validation_test.go#L276

please expand and rename that rather than creating this new function

Reply:
True, it is not duplicating all the exact code, however it is a duplication in intended function.
The PR is making it so there are two separate functions which are creating groups of resources for the test-runner service account. You're introducing a function `ensureTestRunnerRBAC` which does not actually create all the necessary RBAC resources for test-runner.

Whether or not this is a "refactor" is inconsequential, this PR adds additional RBAC resource to be created for the SA. It makes sense to have a function `ensureTestRunnerRBAC()`, but it should create all the resources, including those currently present in `sshSetup()`, which can be removed in favor of this new function

Reply:
Just want to drop a general note, that while we should refrain from refactoring WMCO **product** code as part of a bug fix unless really required, it is fine to refactor **test** code which by definition is not part of the product. Unless of course there is a time crunch involved. IOW, context is important when it comes to making such decisions and not just blindly applying "rules".

Reply:
> True, it is not duplicating all the exact code, however it is a duplication in intended function. The PR is making it so there are two separate functions which are creating groups of resources for the test-runner service account. You're introducing a function `ensureTestRunnerRBAC` which does not actually create all the necessary RBAC resources for test-runner.
> 
> Whether or not this is a "refactor" is inconsequential, this PR adds additional RBAC resource to be created for the SA. It makes sense to have a function `ensureTestRunnerRBAC()`, but it should create all the resources, including those currently present in `sshSetup()`, which can be removed in favor of this new function



Correct, I agree.

Reply:
Addressed in https://github.com/openshift/windows-machine-config-operator/pull/1901/commits/03ce26656619fd8b39712bd55d8d090e51e83cbc